entrypoint.sh

```diff
@@ -27,7 +27,7 @@ else
 fi
 
 if [ -n "${12}" ]; then
-  poetry plugin add ${12}
+  poetry self add ${12}
```

Comment:
Since this action is also supporting using the old versions of poetry, we first need to check which version is currently installed and use `poetry plugin add` with old version and `poetry self add` with new ones. Please add this check if you are in a hurry or wait until next week so I will add it when I come back from the holiday. :)

Reply:
Poetry v1.2.0 stable was just released 🎉 !


I believe `poetry plugin add` was only valid for v1.2.0b1, with `poetry self add` becoming the way in >=v1.2.0b2. So adding a check would really only influence people running v1.2.0b1.

Anyone that updates the version of poetry-publish will also be updating to poetry v1.2.0, so I feel like its not worth the clutter to include a version check.

Enjoy the holidays!

Reply:
Please be a little bit patient, new release is building, it will take some time with my current internet connection :slightly_smiling_face:

Reply:
https://github.com/JRubics/poetry-publish/releases/tag/v1.13